pytests/test_operators.py

```diff
@@ -25,8 +22,8 @@ def add_one(item):
     assert sorted(out) == sorted(
         [
             (0, 1),
```

Comment:
Here and elsewhere in this file I debated the value of checking the epochs at all.